manual/src/refman/extensions/effects.etex

```diff
@@ -43,40 +35,24 @@ We can handle the "Xchg" effect by implementing a handler that always returns
 the successor of the offered value:
 
 \begin{caml_example}{verbatim}
-try_with comp1 ()
-{ effc = fun (type a) (eff: a t) ->
-    match eff with
-    | Xchg n -> Some (fun (k: (a, _) continuation) ->
-        continue k (n+1))
-    | _ -> None }
+try comp1 () with
+| effect (Xchg n), k -> continue k (n+1)
 \end{caml_example}
 
-"try_with" runs the computation "comp1 ()" under an effect handler that handles
-the "Xchg" effect. As mentioned earlier, effect handlers are a generalization
-of exception handlers. Similar to exception handlers, when the computation
-performs the "Xchg" effect, the control jumps to the corresponding handler.
-However, unlike exception handlers, the handler is also provided with the
-delimited continuation "k", which represents the suspended computation between
-the point of "perform" and this handler.
+We run the computation "comp1 ()" under an effect handler that handles the
+"Xchg" effect. As mentioned earlier, effect handlers are a generalization of
```

Comment:
Since this is the introduction of the effect handler syntax, I believe we should expand a bit more on the link between the syntax and the semantic description below.

- What is the effect keyword?
- Why are we matching a pair, what is `k`?

Reply:
> What is the effect keyword?

Added some explanation now. 

> Why are we matching a pair, what is k?

The rest of the highlighted paragraph already talks about what "k" is. 

> Similar to exception handlers, when the computation performs the "Xchg" effect, the control jumps to the corresponding handler, and unhandled effects are forwarded to the outer handler. However, unlike exception 
handlers, the handler is also provided with the delimited continuation "k", which represents the suspended computation between the point of "perform" and this handler.

Is this not sufficient?

Reply:
Fixed in https://github.com/ocaml/ocaml/pull/13295/commits/9306ca3cffd4eac957a6e9acba6ec9990a2f5c18.